utility/BUILD.bazel

```diff
@@ -51,6 +52,7 @@ kt_android_library(
         "//third_party:com_google_guava_guava",
         "//third_party:org_jetbrains_kotlinx_kotlinx-coroutines-core",
         "//utility/src/main/java/org/oppia/android/util/datetime:date_time_util",
+        "//utility/src/main/java/org/oppia/android/util/statusbar:status_bar_color",
```

Comment:
Could you try moving this dependency directly to the app module? I don't see any intradependency on this library in the utility module.